.ci/deploy/devfile-registry.yaml

```diff
@@ -69,6 +69,11 @@ objects:
             limits:
               cpu: 100m
               memory: ${DEVFILE_INDEX_MEMORY_LIMIT}
+          env:
+          - name: ENABLE_TELEMETRY
+            value: true
```

Comment:
This field should be be configurable with a parameter under https://github.com/devfile/registry/blob/b05971c7cf9fc52f280dc23690d25bc01511b479/.ci/deploy/devfile-registry.yaml#L156

And the default value should be false.

Reply:
The parameter will be set in AppSRE's Gitlab instance, where we have the deployment configurations for staging and prod

Reply:
Got it, thanks